docs/reference/release-notes/7.6.asciidoc

```diff
@@ -1,3 +1,82 @@
+[[release-notes-7.6.1]]
+== {es} version 7.6.1
+
+coming[7.6.1]
+
+Also see <<breaking-changes-7.6,Breaking changes in 7.6>>.
+
+[[bug-7.6.1]]
+[float]
+=== Bug fixes
+
+Aggregations::
+* Decode max and min optimization more carefully {pull}52336[#52336] (issue: {issue}52220[#52220])
+* Fix a DST error in date_histogram {pull}52016[#52016] (issue: {issue}50265[#50265])
+
+Audit::
+* Logfile audit settings validation {pull}52537[#52537] (issues: {issue}47038[#47038], {issue}47711[#47711], {issue}52357[#52357])
+
+CCR::
+* Fix shard follow task cleaner under security {pull}52347[#52347] (issues: {issue}44702[#44702], {issue}51971[#51971])
+
+Distributed::
+* Adjust global checkpoint background sync condition {pull}52444[#52444] (issue: {issue}52223[#52223])
+
+Engine::
+* Refresh should not fail already closed engine {pull}51281[#51281] (issue: {issue}48414[#48414])
+
+Features/CAT APIs::
+* fix npe in RestPluginsAction {pull}52620[#52620] (issue: {issue}45321[#45321])
+
+Features/ILM+SLM::
+* ILM fix the init step to actually be retryable {pull}52076[#52076]
+
+Features/Ingest::
+* Handle errors when evaluating if conditions in processors {pull}52543[#52543] (issue: {issue}52339[#52339])
+
+Features/Monitoring::
+* Fix NPE in cluster state collector for monitoring {pull}52371[#52371] (issue: {issue}52317[#52317])
+
+Features/Stats::
+* Switch to AtomicLong for "IngestCurrent" metric to prevent negative values {pull}52581[#52581] (issues: {issue}52406[#52406], {issue}52411[#52411])
+
+Infra/Packaging::
+* Limit _FILE env var support to specific vars {pull}52525[#52525] (issue: {issue}52503[#52503])
+
+Machine Learning::
+* [ML][Inference] don't return inflated definition when storing trained models {pull}52573[#52573]
+* [Ml] Validate tree feature index is within range {pull}52460[#52460]
```

Comment:
We can remove the bracketed content (`[Ml]`, `[ML] [Inference]`) since this is already labelled.

```suggestion
* Son't return inflated definition when storing trained models {pull}52573[#52573]
* Validate tree feature index is within range {pull}52460[#52460]
```

Reply:
I added a commit for this change.  I also added a missing link on the main release notes page.

Reply:
Good catch. Thanks @lcawl.